client/src/components/RatingsReviews/AddReview.jsx

```diff
@@ -142,24 +143,23 @@ export default function AddReview(props) {
 
 
             </div>
-            <label>Review title
-              <input type="text" value={state.summary} name="summary" placeholder="Write your review title here." onChange={handleChange}></input>
-            </label>
+            <label>Review title  <input type="text" value={state.summary} name="summary" placeholder="Write your review title here." onChange={handleChange}></input></label>
+
             <label>Review
               <input type="text" value={state.body} name="body" placeholder="Write your review here." onChange={handleChange}>
               </input></label>
-            <label>Your photos:<textarea value={state.photos} name="photos" onChange={handleChange} rows={5} cols={40} /></label>
+            <label>Your photos:<textarea value={state.photos} name="photos" onChange={handleChange} rows={5} cols={80} /></label>
```

Comment:
just curious - what is rows and cols used for in the textarea?

Reply:
> just curious - what is rows and cols used for in the textarea?

how the box appears.  you can make the textarea appear how you want it to look by setting the initial rows and cols